caffeinated/modules/recaptcha_invisible/EED_Recaptcha_Invisible.module.php

```diff
@@ -150,6 +150,11 @@ public static function protectForm($form)
      */
     public static function localizeScriptVars()
     {
+        /** @var \EventEspresso\core\services\request\Request $request */
+        $request = LoaderFactory::getLoader()->getShared('EventEspresso\core\services\request\Request');
+        if (! ($request->isAdmin() || $request->isFrontend())) {
```

Comment:
I can see this is only localizing the invisible recaptcha vars for EE admin requests and front-end requests (which, I think, means vars aren't localized for non-EE admin requests, REST API requests, ajax requests, and scraping requests). So it's fairly clear what this code is doing, but IMO it would be good to add a PHP comment explaining WHY (eg the invisible recaptcha vars are only used to display the recaptcha, and we only do that on front-end requests and EE admin requests, not on other admin pages or any of the other request types).

Reply:
kk I can do that... 
just to clarify though, the `RequestInterface::isAdmin()` method detects ANY UI request directed to the WP admin, meaning the request satisfies ALL of the following:

- it is NOT a request for ANY of the following:
    - WP Error Scrape (new detection added in this pull)
    - REST API 
    - AJAX
    - WP Cron
    - WP CLI
- a call to `is_admin()` returns `true`

the request context detection works in the order of the above list, so as soon as one of those request types is detected, it tags the request as such and exits. So even though an `is_admin()` check would return `true` for an AJAX request, we check for AJAX first and exit before calling `is_admin()`.